examples/routing_service/routing_service_file_adapter/cpp/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ find_package(
 # Define the library that will provide the adapter plugin
 add_library(
     fileadapter
-    MODULE
+    SHARED
```

Comment:
Why did you change this?

Reply:
Cause it wasn't working on MacOS. It was creating the library as libfileadapter.so. Whereas we were looking for libfileadapter.dylib.

Reply:
Also we support static linking of adapters. It's better to remove the SHARED and indicate in the instructions whether shared libs is on or off

Reply:
Done. Also added additional content in the README.md to describe how the static libraries can be used with RS as a library.